app/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+import "./App.css";
+import { io, Socket } from "socket.io-client";
+import { useState } from "react";
+import Chat from "./Chat";
+
+const socket: Socket = io("http://localhost:3001");
```

Comment:
can you use localhost:3001 from a separate config file/env var instead of using directly